### DISCUSSION TITLE
# of shows in Colosseum?

### DISCUSSION THREAD

**Discussion by catalin-duta** [Keriew/augustus#523] [Q&A] [answered]
How many shows can there be in the Colosseum?

![Screenshot_20211112-090545016](https://user-images.githubusercontent.com/93785920/141424953-98c1e3d9-395e-4661-9e83-0b6409422947.jpg)

It always seems to be the same as the total # of shows in Arenas - oscillating only between 7 and 8 for me because of the Gladiator thing described in my previous post.

**Comment by Keriew** [comment, marked as answer]
Fixed in 6586675
